package.json

```diff
@@ -24,22 +28,42 @@
     "redis",
     "Google Cloud Memorystore for Redis API"
   ],
+  "contributors": [
+    "Chris Wilcox <crwilcox@google.com>"
+  ],
+  "scripts": {
+    "test": "repo-tools test run --cmd npm -- run cover",
+    "generate-scaffolding": "repo-tools generate all",
+    "generate-scaffolding-samples": "repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json",
+    "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader test/*.js && nyc report",
+    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
+    "test-no-cover": "repo-tools test run --cmd mocha -- test/*.js --no-timeouts",
+    "lint": "repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/ smoke-test/",
+    "prettier": "repo-tools exec -- prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js smoke-test/*.js",
+    "docs": "repo-tools exec -- jsdoc -c .jsdoc.js",
+    "system-test": "repo-tools test run --cmd mocha -- system-test/*.js smoke-test/*.js --no-timeouts"
+  },
   "dependencies": {
     "google-gax": "^0.14.2",
     "lodash.merge": "^4.6.0",
     "protobufjs": "^6.8.0"
   },
   "devDependencies": {
-    "mocha": "^3.2.0",
+    "@google-cloud/nodejs-repo-tools": "^2.3.0",
+    "async": "^2.6.0",
+    "codecov": "^3.0.1",
+    "eslint": "^4.19.1",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-node": "^6.0.1",
+    "eslint-plugin-prettier": "^2.6.0",
+    "extend": "^3.0.1",
+    "ink-docstrap": "^1.3.2",
+    "intelli-espower-loader": "^1.0.1",
+    "jsdoc": "^3.5.5",
+    "mocha": "^3.5.3",
```

Comment:
This is an old version of mocha, isn't it?

Reply:
honestly I am unsure. most of this is out of the repo-tools generator (including node 4) I will dig in quick and see and rev this.

Reply:
@alexander-fenster do we have renovate or greenkeeper enabled on repo-tools generator?

Reply:
@JustinBeckwith Nope. The version 3 is what's written in the template. I need to update it (and yes it's manual, there is nothing like greenkeeper as it's not `package.json` but just a template file in some in-house templating language).

Reply:
But Greenkeeper/Renovate will come here and update dependencies for us :)